apis/elbv2/v1beta1/targetgroupbinding_types.go

```diff
@@ -115,6 +115,7 @@ type TargetGroupBindingNetworking struct {
 // TargetGroupBindingSpec defines the desired state of TargetGroupBinding
 type TargetGroupBindingSpec struct {
 	// targetGroupARN is the Amazon Resource Name (ARN) for the TargetGroup.
+	// +kubebuilder:validation:MinLength:=1
```

Comment:
i believe it used to be `kubebuilder:validation:MinLength=1` did `+kubebuilder:validation:MinLength:=1` works as well?

Reply:
It seems both `:=` and `=` output the same manifests and https://book.kubebuilder.io/reference/markers/crd-validation.html uses `:=` instead of `=`. I updated it to use `=` instead!